R/rhtmlCombinedScatter.R

```diff
@@ -399,7 +405,13 @@ CombinedScatter <- function(
     y.title.font.size = 12,
     z.prefix = "",
     z.suffix = "",
-    z.title = "")
+    z.title = "",
+    quadrants.show = FALSE,
+    x.midpoint = 0,
+    y.midpoint = 0,
+    midpoint.line.color = '#000000',
```

Comment:
There should be an `x.midpoint.line.color` and a separate `y.midpoint.line.color`?

Reply:
The story https://numbers.atlassian.net/browse/FS2-2795 does not mention separate line controls for x and y, and the notion page does not mention it either. While this would offer more flexibility, it is also more tedious for users wanting to change both and clutters the UI. It could be something we offer in the future with ngviz point and click.

Reply:
I think its just implied that there are two separate line controls. We have separate line customization controls for x and y on all our axis, grid and zero lines. It seems strange that there would only one set of control for the midpoint lines. If they can customize all 4 quadrants separately, then they should be able to customize the 2 lines separately as well.

Reply:
Hmm OK since we separate it for the other lines (I'm not sure if that was needed either). I don't know the use cases but maybe a user might want separate colors to distinguish them, e.g. red line divides A, blue line divides B. But quadrants will definitely need to be different colors.

Reply:
Yep, the most common use case is probably to have the lines shown the same way. Maybe be we can sync the defaults in the plugins code. But occasionally they'll want something like this: https://umbrex.com/resources/powerpoint-slides/scatter-charts-4-quadrant/

Reply:
How would we sync the defaults without adding additional controls?